lib/IRGen/GenProto.cpp

```diff
@@ -959,6 +959,13 @@ namespace {
     }
 
     void addAssociatedConformance(const AssociatedConformance &req) {
+      if (req.getAssociation()->getASTContext().LangOpts.hasFeature(Feature::Embedded) &&
```

Comment:
It might be better to push this check up to SILWitnessVisitor, which already includes this check for example:

```
        // ObjC protocols do not have witnesses.
        if (!Lowering::TypeConverter::protocolRequiresWitnessTable(requirement))
          continue;
```

Reply:
This will not work because the FragileWitnessTableBuilder needs to pull the witness table entry before returning.